ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ray-cluster-ingress
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/tags: Environment=dev,Team=test
+    # See `ingress.md` for more details about how to choose subnets.
+    alb.ingress.kubernetes.io/subnets: subnet-d9d4ff92, subnet-0c0209db843c9fd72
+    alb.ingress.kubernetes.io/target-type: ip
+    # Health Check Settings
+    alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+    alb.ingress.kubernetes.io/healthcheck-port: traffic-port
+    alb.ingress.kubernetes.io/healthcheck-path: /-/routes
```

Comment:
What's the behavior if this health check fails? Does traffic simply stop flowing to the pod, or does the pod get restarted?

Reply:
yes, traffic will stop sending traffic to the pod. (if there is only one pod which is unhealthy, the traffic will still be sent to that pod.)
No operation on the pod. it still relies on the kuberay operation to take care of it.

Reply:
Sounds good, that makes sense.